log4shell-detector.py

```diff
@@ -40,6 +52,33 @@ def evaluate_log_paths():
     parser.add_argument('--quick', action='store_true', help="Skip log lines that don't contain a 2021 or 2022 time stamp")
     parser.add_argument('--debug', action='store_true', help='Debug output')
     parser.add_argument('--summary', action='store_true', help='Show summary only')
+    parser.add_argument('--check_usage', '-c',action='store_true', help='Check if log4j is being used before launching the scan')
+
+    args = parser.parse_args()
+    
+    print("     __             ____ ______       ____  ___      __          __          ")
```

Comment:
Why is that banner two times in the script? 
Something got mixed up.

Reply:
Fixed